src/query.jl

```diff
@@ -64,9 +64,30 @@ function get_results(f::Float64, voi::VariablesOfInterest, case::Case)::Results
         end
     end
 
+    # These variables will only be in the results if the model has flexible demand
+    # Initialize with empty arrays, to be discarded later if they stay empty
+    load_shift_up = zeros(0, 0)
+    load_shift_dn = zeros(0, 0)
+    try
+        load_shift_up_temp = JuMP.value.(voi.load_shift_up)
+        load_shift_dn_temp = JuMP.value.(voi.load_shift_dn)
+        load_bus_map = sparse(
+            sets.load_bus_idx, 1:sets.num_load_bus, 1, sets.num_bus, sets.num_load_bus
+        )
+        load_shift_up = load_bus_map * load_shift_up_temp
+        load_shift_dn = load_bus_map * load_shift_dn_temp
+    catch e
+        if isa(e, MethodError)
+            # Thrown when storage variables are `nothing`
```

Comment:
`Thrown when storage variables` -> `Thrown when load shift variables`

Reply:
Good catch. That was sloppy of me.

Reply:
Done.